src/shared/metadata/cgroup_path_resolver_test.cc

```diff
@@ -32,6 +32,7 @@ constexpr std::string_view kContainerID =
     "a7638fe3934b37419cc56bca73465a02b354ba6e98e10272542d84eb2014dd62";
 
 TEST(CGroupPathResolver, GKEFormat) {
+  FLAGS_force_cgroup1_mode = true;
```

Comment:
If we must set flags in test, then use `PX_SET_FOR_SCOPE` to make sure it doesn't leak out.